src/components/pages/org/ConnectLink.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { FormattedMessage as Msg } from 'react-intl';
+
+import FormattedLink from '../../../common/misc/FormattedLink';
+
+export default class ConnectLink extends React.Component {
+    render() {
+        const {isAuthenticated, isMember, orgItem} = this.props;
```

Comment:
FYI: We're trying to move away from the term "member", as a user's affiliation with an organization in Zetkin has nothing to do with their membership. This is mostly important in user-facing communication, but it makes sense to move away from it in code as well.